### DISCUSSION TITLE
[IntelliSense] The ability to match the original css syntax property name

### DISCUSSION THREAD

**Discussion by davesierra** [tailwindlabs/tailwindcss#11626] [Ideas]
Stop me if you've heard this or exists, but I tried looking a few mins and could not find it.

Being able to match by the original css style and autocomplete to the tailwind (TW) equivalent would be amazing, especially for beginners still transitioning to TW. I feel this extension is amazing when you already have a general idea of TW's nomenclature (intermediate level?), but it could do a touch more to eliminate having to tab into the TW docs completely with this feature.

An example, I don't know what TW uses for z-index, so I'll type "z-index" and the box should be able to still fuzzy match on all z-index: X which translate to z-X in TW's terms, so I can just select the correct one and it autocompletes with the correct term, and thus I can avoid having to do a tab over to the docs.

A use case I experience, if want to apply a z-index, being still unsure of how TW translates this, I'll think "is it index-1, or zindex-1 or z-index-1". Turns out it was neither and just z-1.

This post is mostly a feeler to see if a feature like this is in plans, or that I maybe could take a crack at implementing it.

**Comment by phegman** [comment]
I agree this would be very useful. Another example is the[ line height utilities](https://tailwindcss.com/docs/line-height) which use `leading-*` naming convention. This may not be super intuitive to someone coming from traditional CSS. If `leading-*` showed up when typing `line-height` it would be very nice!
